ffcli/command.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"text/tabwriter"
 
-	"github.com/peterbourgon/ff/v3"
+	ff "github.com/peterbourgon/ff/v3"
```

Comment:
my current goimports setup requires this, tell me if it's a problem so I can edit this file without IDE hook and come back to the previous state